lib/xapi-stdext-date/date.ml

```diff
@@ -44,50 +45,40 @@ type iso8601 = Ptime.t
   match x |> Ptime.of_rfc3339 |> Ptime.rfc3339_error_to_msg with
   | Error (`Msg e) -> invalid_arg (Printf.sprintf "date.ml:of_string: %s" e)
   | Ok (t, tz, _)  -> match tz with
-                      | None | Some 0 -> t
+                      | None | Some 0 -> (t, UTC)
                       | Some _        -> invalid_arg (Printf.sprintf "date.ml:of_string: %s" x)
```

Comment:
Do you want to keep enforcing this here, or allow `Local`?

Reply:
I'd prefer to not allow `Local` here. I am hoping that the only place we'll ever create a `Local` ptime is in `get_server_localtime`